benchexec/tablegenerator/react-table/scripts/start.js

```diff
@@ -5,8 +5,6 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-"use strict";
```

Comment:
Is removing this intentional?

Reply:
Apparently it's an ESLint recommendation to remove the same, as the "use strict" directive is useless outside of ES modules and does not work for standalone scripts.